server/package.json

```diff
@@ -52,6 +52,7 @@
     "passport-local": "^1.0.0",
     "passport-localapikey-update": "^0.5.0",
     "pushover-notifications": "latest",
+    "request": "^2.88.2",
```

Comment:
Should not be needed. Seemingly by accident, someone required 'request' in api.js, without ever using it - I suspect some IDE's autocomplete that didn't resolve the request param of some function and thought, that a npm lib was to be used.